src/include/rimage/cavs/cavs_ext_manifest.h

```diff
@@ -230,6 +230,7 @@ struct fw_ext_man_cavs_header {
 	uint32_t len;	/* sizeof(Extend Manifest) in bytes */
 	uint16_t version_major;	/* Version of Extended Manifest structure */
 	uint16_t version_minor;	/* Version of Extended Manifest structure */
+	uint16_t version_micro; /* Version of Extended Manifest structure */
```

Comment:
Wont this break manifest parsing on Windows ? @RanderWang can you comment.

Reply:
@lgirdwood Rander is on vacation this week. Let's wait for his comments next week. Hope I'm not breaking Windows.

Reply:
Windows driver will take version_micro as num_module_entries we don't change header file used by windows, and this will result to error.

Reply:
> Windows driver will take version_micro as num_module_entries we don't change header file used by windows, and this will result to error.

This only applies to the extended manifest. So I think we can skip the micro version only for cavs extended manifest.

Reply:
To avoid breaking Windows, I wrote a new PR #92 that will not change manifest header definitions but write the micro version to the unused hotfix_version field of manifest header. Can you help review the new PR#92?